components/button/button-add.js

```diff
@@ -35,6 +37,11 @@ class ButtonAdd extends PropertyRequiredMixin(FocusMixin(LocalizeCoreElement(Lit
 
 	static get styles() {
 		return css`
+			:host {
+				--d2l-button-add-animation-delay: 50ms;
+				--d2l-button-add-animation-duration: 200ms;
+				--d2l-button-add-hover-focus-color: var(--d2l-color-celestine-minus-1);
```

Comment:
Looking at the diffs... the focus color, at least of the border around the text/icon seems darker than our normal buttons. Is this expected?

Reply:
This is expected - since we are using celestine already for the icon + text case, we wanted to use a different colour for the hover/focus state